### DISCUSSION TITLE
Add Icons to Admin Menu entries

### DISCUSSION THREAD

**Discussion by jsalgado10** [OrchardCMS/OrchardCore#8358] [Q&A]
Is there a way to add icons to an admin menu item programmatically?

I am able to add the items to the menu, but would like to add an icon. I see that using .AddClass method seems to be adding the icon but i am not sure what classes are available.

Thanks

**Comment by TaylorEdm** [comment]
They are using fontawesome for icons and you can search for it [here](https://fontawesome.com/icons?d=gallery&m=free):

**Reply by hishamco** [reply]
I think there is an icon picker that can expose all the icons or use the link that @TaylorEdm mentioned

**Reply by jsalgado10** [reply]
I am using an AdminMen class on my module to create the  new menu items. This is what I currently have

            builder
                .Add(S["Notifications"], S["Notifications"].PrefixPosition(), configuration => configuration
                    .AddClass("fa-bell").Id("notifications")
                         .Add(S["Settings"], S["Settings"].PrefixPosition(), entry => entry
                         .AddClass("fa-cogs").Id("settings")
                               .Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = NotificationsSettingsDisplayDriver.GroupId })
                               .Permission(Permissions.ManageNotificationsSettings)
                               .LocalNav())
                         .Add(S["Dashboard"],S["Dashboard"].PrefixPosition(), entry => entry
                         .AddClass("fa-tachometer-alt").Id("dashboard")
                            .Url("~/Notifications/Dashboard")
                            .Permission(Permissions.ManageNotificationsSettings)
                            .LocalNav())
                 );

If I set the class to "fa fa-bell" I get  a question mark instead of the menu.

**Reply by hishamco** [reply]
Just use "fa-bell"

**Reply by jsalgado10** [reply]
i got it working.

I had to set the class using icon-class then the fontawesome class
.AddClass("icon-class-fa-bell").AddClass("icon-class-fas").Id("notifications")
